UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/RNWTestHelper.cs

```diff
@@ -0,0 +1,174 @@
+﻿using FakeItEasy;
+using Microsoft.AspNetCore.Http;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using radioNavigationalWarningDto = UKHO.MaritimeSafetyInformation.Common.Models.RadioNavigationalWarning.DTO;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using UKHO.MaritimeSafetyInformation.Common;
+using UKHO.MaritimeSafetyInformation.Common.Configuration;
+using UKHO.MaritimeSafetyInformation.Web.Services;
+
+namespace UKHO.MaritimeSafetyInformation.IntegrationTests.RadioNavigationalWarning
+{
+    public class RNWTestHelper
+    {
+        public readonly RadioNavigationalWarningsContext _fakeContext;
+        public IOptions<RadioNavigationalWarningConfiguration> _fakeRadioNavigationalWarningConfiguration;
+        public IHttpContextAccessor _fakeHttpContextAccessor;
+        public ILogger<RNWRepository> _fakeLoggerRnwRepository;
```

Comment:
Please remove unused variable _fakeLoggerRnwRepository

Reply:
Changes are done as requestee, Thank You.